src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -32,8 +32,7 @@
     @JsonIgnore
     private final List<ValidationStrategyConfiguration> validationStrategies = Lists.newArrayList();
 
-    @Valid
-    private PartitionStrategyDescriptor partitionStrategy = RANDOM_PARTITION_STRATEGY;
+    private String partitionStrategy = PartitionStrategy.RANDOM_STRATEGY;
```

Comment:
I'm not sure, but if you make it an `Enum` instead of a `String` wouldn't it be validated by default?

I mean, if the user provides a value that is not in the `enum` then it would fail. I have to check, but if yes, then it would remove the need to validate manually.